getstarted.rst

```diff
@@ -1,77 +1,119 @@
 Getting Started
 ===============
 
+.. _requirements:
+
 Requirements
 ------------
 
 * `Python 3.4 or greater <https://www.python.org/download/>`_
 * `pip <https://pip.pypa.io/en/latest/installing.html>`_
-* virtualenv - ``pip install virtualenv``
+* `virtualenv <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_
+* `git <http://git-scm.com/download/mac>`_
+
+First, open a terminal in to the home directory and check that you have everything you need.
+
+.. code-block:: bash
+
+    python3 --version
+    pip3.4 --version
+    virutalenv --version
+    git --version
+
+If those four commands don't return anything, then follow the requirements links before continuing.
+
+System Setup
+------------
 
+As a standard, we will be installing each version of nio to its own virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them `here <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+
+Each nio project will be in its own directory too. We will start by creating some directories for these.
+
+.. code-block:: bash
+
+    cd
+    mkdir nio
+    cd nio
+    mkdir projects
+    mkdir versions
 
 Installation
 ------------
 
-We will be installing nio to a virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them at `here <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+Lets start by installing nio version 1.5.1.
 
 .. code-block:: bash
 
-    $ mkdir nio
-    $ cd nio
-    $ virtualenv env
-    $ source env/bin/activate
+    cd ~/versions
+    virtualenv-3.4 1.5.1
+
+If virtualenv was installed with a python version less than 3.4, then you will need to specify the python version when creating your virtual environtment. To check which version of python virtualenv will use, type `virtualenv -h` and look at what version is specifed in the `-p` section of the docs.
```

Comment:
I would take this out and just do `virtualenv -p python3` by itself. If they are a sysadmin or want to learn what these commands do, they can look it up themselves! This guide is supposed to remain as simple as possible, and this is as simple as it can get :D